src/program.js

```diff
@@ -103,7 +103,9 @@ export class Program {
       await runCommand(argv);
     } catch (error) {
       const prefix = cmd ? `${cmd}: ` : '';
-      log.error(`\n${prefix}${error.stack}\n`);
+      if (!error instanceof UsageError) {
```

Comment:
The coverage report says the `log` statement within this if branch isn't covered but we definitely have a test that covers it. This makes me think the JS interpreter is parsing `!error` before passing the value to `instanceof`. Try this instead and let's see if the code coverage goes back to 100%:
````javascript
if (!(error instanceof UsageError)) {
}
````